packages/config/src/test/projects.test.ts

```diff
@@ -21,19 +22,60 @@ describe('projects', () => {
           const contracts = project.contracts?.addresses ?? []
           for (const contract of contracts) {
             testAddress(contract.address)
-            if (
-              contract.upgradeability?.type === 'EIP1967' ||
-              contract.upgradeability?.type === 'NutBerry' ||
-              contract.upgradeability?.type === 'ZeppelinOs'
-            ) {
-              testAddress(contract.upgradeability.admin)
-              testAddress(contract.upgradeability.implementation)
-            }
-            if (contract.upgradeability?.type === 'StarkWare') {
-              testAddress(contract.upgradeability.implementation)
-              if (contract.upgradeability.callImplementation) {
-                testAddress(contract.upgradeability.callImplementation)
+            if (!contract.upgradeability?.type) return
```

Comment:
If we're using `EthereumAddress` in config then this entire test is no longer needed